lib/loga/railtie.rb

```diff
@@ -153,26 +153,47 @@ def self.call
 
       def call
         ensure_subscriptions_attached
+        notify_for_delivered_message
         remove_log_subscriptions
       end
 
       private
 
+      def notify_for_delivered_message
+        ActiveSupport::Notifications.subscribe('deliver.action_mailer') do |*args|
```

Comment:
It would be better if you write a custom `LogSubscriber`
https://github.com/rails/rails/blob/v5.2.0/actionmailer/lib/action_mailer/log_subscriber.rb

Reply:
I was writing the exact same comment 😂 

> Have you considered extracting the implementation to a separate class, perhaps in `lib/loga/log_subscribers/action_mailer.rb` (similarly to [here](https://github.com/rails/rails/blob/9575f4fdca7d3bc4b1ded9d49d6c127eee96b4a4/actionmailer/lib/action_mailer/log_subscriber.rb))? We could then attach our custom log subscriber in this method: 
```ruby
def notify_for_delivered_message
   LogSubscribers::ActionMailer.attach_to :action_mailer
end
```